tests/schemas/test_coreapi.py

```diff
@@ -1214,13 +1214,13 @@ def test_from_router(self):
                     'detail_export': coreapi.Link(
                         url='/from-routercollision/detail/export/',
                         action='get',
-                        description=desc_0)
-                },
-                'detail_0': coreapi.Link(
-                    url='/from-routercollision/detail/',
-                    action='get',
-                    description=desc_1
-                )
+                        description=desc_0),
+                    'detail': coreapi.Link(
```

Comment:
I'm not entirely sure I agree with this being the right location for this link. This seems like it should represent `/from-routercollision/detail/detail/`, but instead it is representing one layer up.

Additionally, this breaks backwards compatibility with how we handle naming collisions. (See #5486 for where this was first implemented)